prow/cmd/deck/static/api/prow.ts

```diff
@@ -1,15 +1,38 @@
 export type JobType = "presubmit" | "postsubmit" | "batch" | "periodic";
 export type JobState = "triggered" | "pending" | "success" | "failure" | "aborted" | "error" | "unknown" | "";
 
-export interface Job {
-  type: JobType;
+// Pull describes a pull request at a particular point in time.
+// Pull mirrors the Pull struct defined in types.go.
+export interface Pull {
+  number: number;
+  author: string;
+  sha: string;
+  title: string;
+  ref: string;
+  link: string;
+  commit_link: string;
+  author_link: string;
+}
+
+// Refs describes how the repo was constructed.
+// Refs mirrors the Refs struct defined in types.go.
+export interface Refs {
+  org: string;
```

Comment:
As above, these are all `omitempty` and should be marked as optional using question marks (e.g. `org?: string`).